requirements.txt

```diff
@@ -1,9 +1,9 @@
-aiofiles>=0.8.0
-async_lru~=1.0
-galaxy_importer>=0.4.5
-GitPython>=3.1.24
-jsonschema~=4.4
-packaging
-pulpcore>=3.18
-PyYAML
-semantic_version
+aiofiles>=0.8.0,<0.9
+async_lru>=1.0,<1.1
+galaxy_importer>=0.4.5,<0.5
+GitPython>=3.1.24,<3.2
+jsonschema>=4.4,<4.7
+packaging>=21.3,<22
+pulpcore>=3.20
```

Comment:
I think, with the new deprecation policy, we should specify `pulpcore>=3.20,<3.25` here. The reason we kept this without the upper bound was that we had to change it with every pulpcore release. And that very reason is addressed by the new policy.

Reply:
I agree. @mdellweg how can we identify this more broadly for all plugin writers?

Reply:
We could move the upper bounds check from the release pipeline to the main ci.

Reply:
We should do this @dkliban @fao89 FYI. Basically we want to address release-time issues ahead of release time FTW.

Reply:
it is a matter of calling https://github.com/pulp/pulp_ansible/blob/main/.ci/scripts/upper_bound.py at lint time

Reply:
@fao89 how would you feel about making that PR?

Reply:
I can do it, but people need to be aware as it would break all the CIs

Reply:
https://github.com/pulp/plugin_template/pull/666